metricflow/test/snapshots/test_dataflow_to_sql_plan.py/SqlQueryPlan/PostgresSqlClient/test_cumulative_metric__plan0.sql

```diff
@@ -1,31 +1,33 @@
 -- Compute Metrics via Expressions
 SELECT
-  subq_5.txn_revenue AS trailing_2_months_revenue
-  , subq_5.ds__month
+  subq_4.txn_revenue AS trailing_2_months_revenue
```

Comment:
I think there's a bug in how cumulative metrics are generated with `metric_time` - pending further investigation.

Reply:
Actually, this is because the test is not using `metric_time`:

https://github.com/transform-data/metricflow/blob/1c498ddf147eb3e1d9d3b6fe983425824e9af459/metricflow/test/plan_conversion/test_dataflow_to_sql_plan.py#L769

Although it seems like we should throw an error if you try to select a cumulative metric with a window / grain to date without it.

Reply:
That's probably reasonable. I guess the question is if we should throw an error if they manually select the true column `metric_time` would target - like I'm assuming `ds` in this case is what `metric_time` would resolve to in practice.

Let's file a separate issue for that so we can get the Postgres work unstuck.